kombu/transport/redis_cluster.py

```diff
@@ -296,7 +298,13 @@ def conn_or_acquire(self, client=None):
     def _create_client(self, asynchronous=False):
         conninfo = self.connection.client
 
-        return RedisClusterConnection.get_connection(conninfo.hostname, conninfo.port)
+        hostname = conninfo.hostname
+        port = conninfo.port
+        password = conninfo.password
+        transport = self.connection.client.transport_cls
+        ssl = transport == 'redis-clusters'
+
+        return RedisClusterConnection.get_connection(hostname, port, password, ssl)
```

Comment:
What is the purpose of `RedisClusterConnection` class, it looks like it holds `RedisCluster` with `(host, port)` key but we really need this cache? Because `RedisClusterChannel` use its cached `client` when `conn_or_acquire` is called

Reply:
I'm removing it on https://github.com/sendbird/kombu/pull/8